test/__snapshots__/index.spec.js.snap

```diff
@@ -840,14 +840,14 @@ Array [
     "severity": 2,
   },
   Object {
-    "column": 5,
```

Comment:
Do you know why was this snapshot changed? Seems unrelated.

Reply:
I was having trouble rebuilding the snapshot file within my windows client, so i had to change to ubuntu.
I will definitly take a look at it and report.

Reply:
A fresh install of node fixed the issue, now building as expected.